pkg/models/datasource_cache.go

```diff
@@ -332,3 +335,16 @@ func awsServiceNamespace(dsType string) string {
 		panic(fmt.Sprintf("Unsupported datasource %q", dsType))
 	}
 }
+
+// newConntrackDialContext takes a http.DefaultTransport and adds the Conntrack Dialer
+// so we can instrument outbound connections
+func newConntrackDialContext(name string) func(context.Context, string, string) (net.Conn, error) {
+	return conntrack.NewDialContextFunc(
+		conntrack.DialWithTracing(),
```

Comment:
I don't see how we can make use of this, hence maybe remove it?

Reply:
May be useful for tracing, but we can remove that as you say and maybe re-add it if needed.